lib/snapshot-manager.js

```diff
@@ -355,7 +367,16 @@ class Manager {
 	}
 }
 
-function determineSnapshotDir(projectDir, testDir) {
+function determineSnapshotDir(options) {
+	const projectDir = options.projectDir;
+	const testDir = determineSourceMappedDir(options);
+	if (options.location) {
+		// "snapshotLocation" in ava package.json config, snapshots derive a location
+		// relative from the source testDir not the preset "__snapshots__" folder names
```

Comment:
Good point, I guess the comment highlighted some of the ambiguitiy, please checkout the latest revision.